packages/trader/src/AppV2/Components/Guide/decription-modal.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { ActionSheet, Heading, Chip, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { AVAILABLE_CONTRACTS, CONTRACT_LIST } from 'AppV2/Utils/trade-types-utils';
+import TradeDescription from './trade-description';
+
+type TDescriptionModal = {
+    is_open?: boolean;
+    onClose: () => void;
+    onChipSelect: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+    onTermClick: (term: string) => void;
+    selected_contract_type: string;
+};
+
+const DescriptionModal = ({
+    is_open,
+    onClose,
+    onChipSelect,
+    onTermClick,
+    selected_contract_type,
+}: TDescriptionModal) => {
+    //TODO: temporary, until we'll have ordered list, coming from contract type selection
+    const order = [
+        CONTRACT_LIST['RISE/FALL'],
+        CONTRACT_LIST.ACCUMULATORS,
+        CONTRACT_LIST.MULTIPLIERS,
+        CONTRACT_LIST.VANILLAS,
+        CONTRACT_LIST.TURBOS,
+        CONTRACT_LIST['HIGHER/LOWER'],
+        CONTRACT_LIST['TOUCH/NO TOUCH'],
+        CONTRACT_LIST['MATCHES/DIFFERS'],
+        CONTRACT_LIST['EVEN/ODD'],
+        CONTRACT_LIST['OVER/UNDER'],
+    ];
+
+    const ordered_contract_list = [...AVAILABLE_CONTRACTS].sort(
```

Comment:
I wanted to use `toSorted` (as it created a new array), but it was not supported by our version of TS . Hence, used `spread` and `sort` :(